applications/nrf_desktop/configuration/nrf52820dongle_nrf52820/child_image/mcuboot/prj_release.conf

```diff
@@ -33,6 +33,7 @@ CONFIG_UART_INTERRUPT_DRIVEN=y
 
 # MCUBoot serial
 CONFIG_GPIO=y
+CONFIG_GPIO_NRFX_INTERRUPT=n
```

Comment:
Consider migrating to this configuration with all MCUboot projects in the nRF Desktop as we don't use this feature. We should aspire to have uniform behaviour across all targets as we are unable to manually validate the functionality with every target

Reply:
The GPIO driver is used only for configurations that enable MCUboot serial recovery. Did I overlook any of these configurations?

Reply:
Oh, I see. I wasn't aware that only the serial recovery configurations use it. In that case, I have no comments here